### DISCUSSION TITLE
Next Version Release Date

### DISCUSSION THREAD

**Discussion by jvnvenu** [ClosedXML/ClosedXML#2367] [Q&A]
Hi Team, we are waiting for the latest version which uses OpenDocument.XML 3.0.0 or latest. Please share the release date. It will be helpful if it is soon.

**Comment by jvnvenu** [comment]
Hi team any update?

**Comment by swegele** [comment]
Looking forward to the latest release for same reason.

**Comment by smolleman** [comment]
Same for us, OpenXML v3 seems to fix one of our issues. The milestone seems to be done?
https://github.com/ClosedXML/ClosedXML/milestone/26
Is there anything holding the release back?

**Comment by dluc** [comment]
A fix would be really appreciated :-)

Parsing an Excel file with ClosedXML 0.104.0-preview2 + OpenXml to 3.1.0:
```
Unhandled exception. System.MissingMethodException: Method not found: 'DocumentFormat.OpenXml.OpenXmlElementList DocumentFormat.OpenXml.OpenXmlElement.get_ChildElements()'.
   at ClosedXML.Excel.XLWorkbook.LoadSpreadsheetDocument(SpreadsheetDocument dSpreadsheet)
   at ClosedXML.Excel.XLWorkbook.LoadSheets(Stream stream)
   at ClosedXML.Excel.XLWorkbook.Load(Stream stream)
   at ClosedXML.Excel.XLWorkbook..ctor(Stream stream, LoadOptions loadOptions)
   at ClosedXML.Excel.XLWorkbook..ctor(Stream stream)
```

**Comment by maartenoosterhoff** [comment]
An update would be appreciated 👍

**Comment by dluc** [comment]
FYI version `0.104.0-rc1` works fine with OpenXml `3.1.0`

```
<PackageVersion Include="ClosedXML" Version="0.104.0-rc1" />
<PackageVersion Include="DocumentFormat.OpenXml" Version="3.1.0" />
```

Thanks for releasing the update!
